plasmapy/physics/parameters.py

```diff
@@ -1104,12 +992,7 @@ def Debye_length(T_e, n_e):
     """
 
     T_e = T_e.to(u.K, equivalencies=u.temperature_energy())
-
-    try:
-        lambda_D = ((eps0 * k_B * T_e / (n_e * e ** 2)) ** 0.5).to(u.m)
-    except Exception:
-        raise ValueError("Unable to find Debye length.")
-
+    lambda_D = np.sqrt(eps0 * k_B * T_e / (n_e * e ** 2)).to(u.m)
```

Comment:
Move the `to(u.m)` to the return line as this seems to be the standard in other functions.